config/default_configs/default_edmf_config.yml

```diff
@@ -18,7 +18,10 @@ edmfx_detr_model:
   help: "EDMFX detrainment closure.  [`nothing` (default), `PiGroups`, `Generalized`, `GeneralizedHarmonics`]"
   value: ~
 edmfx_upwinding:
-  help: "EDMFX upwinding mode [`none` (default), `first_order` , `third_order`, `boris_book`, `zalesak`]"
+  help: "EDMFX upwinding mode [`none` (default), `first_order`, `third_order`, `boris_book`, `zalesak`]"
+  value: none
+edmfx_sgsflux_upwinding:
+  help: "EDMFX SGS mass flux upwinding mode [`none` (default), `first_order`, `third_order`, `boris_book`, `zalesak`]"
```

Comment:
I think the `zalesak` didn't compile for me what I tried to use it in EDMF. Maybe we should just suggest the choice between 1st and 3rd order?